src/test/scala/uk/gov/hmrc/perftests/example/Simulation.scala

```diff
@@ -21,99 +21,99 @@ import uk.gov.hmrc.perftests.example.Requests._
 
 class Simulation extends PerformanceTestRunner {
 
-  setup("start-page-business", "Start Page - Business") withRequests (getStartPage, postStartPageBusiness)
+  setup("start-page-business", "Start Page - Business").withRequests(getStartPage, postStartPageBusiness)
```

Comment:
there was a new deprecation warning on using infix notation in these places which I fixed